plugins/frontend-app/.toolkitrc.yml

```diff
@@ -1,3 +1,8 @@
 plugins:
   - '@dotcom-tool-kit/webpack'
   - '@dotcom-tool-kit/backend-app'
+
+hooks:
+  'run:local':
+    - Node
+    - WebpackWatch
```

Comment:
**thought**: don't we want the package to have been built (once) with webpack before we run it with Node?

Reply:
hmm, yes, so; the webpack-built files aren't necessary for the server to run, they're just served alongside the node server usually. except, they actually are required in most cases, since the Page Kit asset loader tries to load the manifest generated by webpack on initialisation. i guess there's two potential solutions:

- add a `WebpackDevelopment` task here that runs before `Node`
- have `dotcom-server-asset-loader` lazily load the manifest

the second one seems hard so